UKHomeOffice/nmsw-ui
# NMSW-470 p2 Refactor declarationId from state to params - task detail and file upload confirmation

# Ticket

NMSW-470

----

## AC

part 2, see https://github.com/UKHomeOffice/nmsw-ui/pull/269

----

## To test

- click the general declaration link
- > you should be taken to the general declaration page with the declaration id in the url
- upload a different general declaration file with a different ship name
- click check for errors
- click save and continue
- > you should be returned to the task details page
- > you should see the new ship name in the task details page

----

## Notes
The following pages will not work as they've not yet been updated
- crew
- passenger
- supporting docs
- CYA
- delete
